test/webpack.config.js

```diff
@@ -47,6 +47,7 @@ const TEST_CONFIG = Object.assign({}, COMMON_CONFIG, {
     stats: {
       warnings: false
     },
+    contentBase: resolve(__dirname, '../modules'),
```

Comment:
I recently added an alias system that allows us to use the same alias definitions and paths to read data files from the webpack dev server that we use to reference code files in requires, [example](https://github.com/uber-web/loaders.gl/blob/master/modules/ply/test/ply-loader.spec.js#L33).

I prefer we converge towards one system (including the `arraybuffer-loader` setup we have at least 3 different systems right now), and I had just started to align the test code to use the new alias system, so I would prefer to either drop this change for now, or sync and agree on best approach.

PS - This is the simple alias system:
* https://github.com/uber-web/loaders.gl/blob/master/modules/core/src/read-file/file-aliases.js

Reply:
How is the example going to work with dist test (one level deeper than src)?

Reply:
This is how it looks in test cases now https://github.com/uber-web/loaders.gl/blob/master/modules/ply/test/ply-loader.spec.js#L33

Since the webpack dev server is run from the root directory, everything inside the repo should be accessible.

Reply:
If we're testing `dist`, then `@loaders.gl/ply` will be replaced with `@loaders.gl/ply/dist/es5` and the above link will not be resolved correctly.